cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/core/audit/AuditProperties.java

```diff
@@ -0,0 +1,105 @@
+package org.apereo.cas.configuration.model.core.audit;
+
+import org.apereo.cas.configuration.model.support.jpa.AbstractJpaProperties;
+import org.apereo.inspektr.audit.support.AbstractStringAuditTrailManager;
+
+/**
+ * This is {@link AuditProperties}.
+ *
+ * @author Misagh Moayyed
+ * @since 5.0.0
+ */
+public class AuditProperties {
+
+    private String appCode = "CAS";
+
+    private String singlelineSeparator = "|";
+
+    private boolean useSingleLine;
+
+    private Jdbc jdbc = new Jdbc();
+
+    private AbstractStringAuditTrailManager.AuditFormats auditFormat =
+            AbstractStringAuditTrailManager.AuditFormats.DEFAULT;
+
+    private boolean ignoreAuditFailures;
+
+    public Jdbc getJdbc() {
+        return jdbc;
+    }
+
+    public void setJdbc(final Jdbc jdbc) {
+        this.jdbc = jdbc;
+    }
+
+    public String getAppCode() {
+        return appCode;
+    }
+
+    public void setAppCode(final String appCode) {
+        this.appCode = appCode;
+    }
+
+    public String getSinglelineSeparator() {
+        return singlelineSeparator;
+    }
+
+    public void setSinglelineSeparator(final String singlelineSeparator) {
+        this.singlelineSeparator = singlelineSeparator;
+    }
+
+    public boolean isUseSingleLine() {
+        return useSingleLine;
+    }
+
+    public void setUseSingleLine(final boolean useSingleLine) {
+        this.useSingleLine = useSingleLine;
+    }
+
+    public AbstractStringAuditTrailManager.AuditFormats getAuditFormat() {
+        return auditFormat;
+    }
+
+    public void setAuditFormat(final AbstractStringAuditTrailManager.AuditFormats auditFormat) {
+        this.auditFormat = auditFormat;
+    }
+
+    public boolean isIgnoreAuditFailures() {
+        return ignoreAuditFailures;
+    }
+
+    public void setIgnoreAuditFailures(final boolean ignoreAuditFailures) {
+        this.ignoreAuditFailures = ignoreAuditFailures;
+    }
+
+    public static class Jdbc extends AbstractJpaProperties {
+        private int maxAgeDays = 180;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['180' is a magic number.](https://www.codacy.com/app/mmoayyed/cas/file/2768422514/issues/source?bid=3256115&fileBranchId=3357337#l76)